lib/pages/login_page.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:ui';
+
+import 'package:email_validator/email_validator.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class LoginPage extends StatefulWidget {
+  LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: Container(
+            constraints: BoxConstraints.expand(),
```

Comment:
I guess you are trying to expand `Container` to be the same size as `Center`, don't I? 🤔  If so, we don't need to set it explicitly because `Container` size get its own constraints from its parent `Center` by default. Please refer [here](https://flutter.dev/docs/development/ui/layout/constraints) for more details

Reply:
I think there was an issue before with the button, it was wrap_content somehow 😂  I put this in to stretch all the children to expand...removing this also works now so I will update this...